test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -261,7 +261,7 @@ def test_resnet18(self):
         # So we are explicitly calling `model.eval()` for any model that contains
         # batch norm.
         # Ref: https://github.com/pytorch/pytorch/issues/99662#issuecomment-1528178221
-        model = torchvision.models.resnet18(pretrained=False).eval()
+        model = torchvision.models.resnet18(weights=None).eval()
```

Comment:
maybe for those we can just leave the default? No strong opinion though

```suggestion
        model = torchvision.models.resnet18().eval()
```

Reply:
`pretrained=False` was the default, but still it was specifically provided for some reason (probably to be explicit - because it's not super clear what's the default if you don't know). So I think it's better to explicitly provide the default value in the new API also.